### PR TITLE
Recent usage indicator on index page

### PR DESCRIPTION
**Pull request by fredex42**
- turning the usage indicator red if it's recent on the index page as well
- modifying tool tip to explain why the indicator is red

### CODE REVIEW

**Comment by fredex42** [comment]
redfront

File: kahuna/public/js/preview/image.js [hunk 1]
```diff
@@ -59,10 +59,14 @@ image.controller('uiPreviewImageCtrl', [
     const hasDigitalUsages$ =
         imageUsagesService.getUsages(ctrl.image).hasDigitalUsages$;
 
+    const usageListRecent$ = imageUsagesService.getUsages(ctrl.image).usageListAfter$(imageUsagesService.recentTime);
+
+        console.log("usageListRecent$: " + usageListRecent$)
```
Comment (clloyd) [inline kahuna/public/js/preview/image.js]: Can we remove the console.log

File: kahuna/public/js/services/image/usages.js [hunk 2]
```diff
@@ -42,6 +42,17 @@ imageUsagesService.factory('imageUsagesService', [function() {
             const hasPlatformUsages = (platform) =>
                 filterByPlatform(platform).every((group) => !group.isEmpty());
 
+            const usageListAfter$ = (since) => usages$.map((usagesList) => {
+                    const nowtime = new Date();
+                    return usagesList.filter((usage) => {
+                        // console.log("evaluating for " + usage);
```
Comment (clloyd) [inline kahuna/public/js/services/image/usages.js]: Again can we remove comments/console.logs

File: usage/app/lib/Config.scala [hunk 3]
```diff
@@ -85,7 +85,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
       .withRegion(Regions.EU_WEST_1)
 
   val postfix = if (stage == "DEV") {
-    iamClient.getUser().getUser().getUserName()
+    "ropnert"
```
Comment (clloyd) [inline usage/app/lib/Config.scala]: Not sure what we decided for this, but shouldn't be in this PR

File: kahuna/public/js/preview/image.js (hunk 1 above)
Comment (fredex42) [inline kahuna/public/js/preview/image.js]: doh! will do

File: usage/app/lib/Config.scala (hunk 3 above)
Comment (fredex42) [inline usage/app/lib/Config.scala]: agreed!

Comment (fredex42) [inline usage/app/lib/Config.scala]: again needed to put something temporary in so that it would work. discussion is in seperate PR.

**Comment by fredex42** [comment]
now release :)

**Comment by paperboyo** [comment]
Hello! This is not working correctly (is out-of-sync with Usage indicators on a picture page). If there is no way to fix it, I would get rid of it as safe images are marked as unsafe...

**Pull request merged**